opta/commands/apply.py

```diff
@@ -130,6 +130,42 @@ def _apply(
     else:
         raise Exception(f"Cannot handle upload config for cloud {layer.cloud}")
 
+    service_modules = (
+        layer.get_module_by_type("k8s-service")
+        if layer.cloud == "aws"
+        else layer.get_module_by_type("gcp-k8s-service")
+    )
+
+    if len(service_modules) > 0 and (
+        Terraform.downloaded_state.get(layer.name, False)
+        or Terraform.download_state(layer)
+        and get_cluster_name(layer) is not None
+    ):
+        configure_kubectl(layer)
+
+        for service_module in service_modules:
+            current_tag = current_image_tag(layer)
+            if (
+                current_tag is not None
+                and image_tag is None
+                and service_module.data.get("image", "") == "AUTO"
+                and not test
+            ):
+                response = (
+                    True
+                    if auto_approve
```

Comment:
I thought we said auto_approve won't apply in this situation

Reply:
I thought we said it would?

Reply:
Naah. I don’t think we should allow users to kill their pods with auto approve. That has to be a manual operation.